assignments/objects.js

```diff
@@ -18,28 +18,69 @@ const example = {
 }
 
 // Write your intern objects here:
+const obj1 = {
+  "id": 0,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const obj2 = {
+  "id": 1,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
 
+const obj3 = {
+  "id": 2,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+
+const obj4 = {
+  "id": 3,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
 
+const obj5 = {
+  "id": 4,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+}
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(obj1["name"])
 // Kennan's ID
-
+console.log(obj2["id"])
 // Keven's email
-
+console.log(obj3["email"])
 // Gannie's name
-
+console.log(obj4["name"])
 // Antonietta's Gender
-
+console.log(obj5["gender"])
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
+function speak () {
+  console.log("Hello, my name is" + this["name"])
+}
+obj2.speak ()
```

Comment:
We declared the speak function, but we didn't give it to `obj2`. We can assign it as a property of an object just like any other value, with either `obj2.speak = speak` or `obj2["speak"] = speak`. And the we can access and call it with `obj2.speak()`.